install_script.sh.template

```diff
@@ -1140,8 +1174,12 @@ if [ "$OS" == "RedHat" ]; then
       done
     fi
 
+    # Install the installer
+    install_installer "$sudo_cmd" "$OS" "$DISTRIBUTION" "$DD_INSTALLER" "$DD_APM_INSTRUMENTATION_ENABLED" || true
```

Comment:
nit, having both in the same function, embedding telemetry of the packages filtering would be nice

ex on the install span
- packages_pre_filter
- packages_filtered

Reply:
yeah that's a good one, added it